contracts/superclasses/Posts.sol

```diff
@@ -82,15 +83,20 @@ contract Posts is KetlGuarded {
   mapping(uint => mapping(uint => Post[])) public comments;
   mapping(uint => mapping(uint => Counters.Counter)) public lastCommentIds;
 
-  mapping(uint => mapping(uint => mapping(uint => Reaction[])))
+  mapping(uint => mapping(uint => mapping(uint => mapping(uint => Reaction))))
```

Comment:
This change is unnecessary

Reply:
Not sure, right now this map works like:
`reactions[feedId][postId][commendId][reactionId]`
If `commentId === 0`, you're accessing the post

`Reaction` has `reactionId` inside, if you'll remove something from reaction array, you'll need to update every `reactionId`
Also you need to make sure on frontend that `currentUserReaction` is proper (because you removed reaction, array has shifted and you need to shift it on frontend as well)